docs/src/components/ColorSwatchFamilyList/ColorSwatchFamilyList.component.tsx

```diff
@@ -0,0 +1,82 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import classNames from 'classnames';
+import { ColorSwatchFamily } from '../ColorSwatchFamily';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyListProps {
+  className?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamilyList = styled.div`
+
+`;
+
+
+// JSX MARKUP
+export const ColorSwatchFamilyList = ({
+  className
+}: ColorSwatchFamilyListProps): React.ReactElement<any> => {
+
+  const [copyChoice, setCopyChoice] = useState('hexCode');
+
+  return (
+    <StyledColorSwatchFamilyList
+      className={classNames(className)}
+    >
+
+      <p> Each color swatch has 4 pills representing the color contrast when used on these colors:</p>
```

Comment:
If you import Typography this page will be a lot easier to deliver, unless I did a poor job building that component, in which case let's make it better!